@xen-orchestra/lite/src/components/pool/dashboard/PoolDashboardNetworkChart.vue

```diff
@@ -100,11 +99,27 @@ const isStatFetched = computed(() => {
 
 const isLoading = computed(() => isFetching.value || !isStatFetched.value)
 
-// TODO: improve the way to get the max value of graph
-// See: https://github.com/vatesfr/xen-orchestra/pull/6610/files#r1072237279
-const customMaxValue = computed(
-  () => Math.max(...map(data.value[0].data, 'value'), ...map(data.value[1].data, 'value')) * 1.5
-)
+const customMaxValue = computed(() => {
+  const values = data.value.reduce(
+    (acc, series) => [...acc, ...series.data.map(item => item.value ?? 0)],
+
+    [] as number[]
+  )
+
+  if (values.length === 0) {
+    return 100
+  }
+
+  const maxUsage = Math.max(...values) * 1.2
+
+  return Math.ceil(maxUsage / 100) * 100
+})
+
+const customValueFormatter = (value: number | null) => {
+  if (value === null) {
+    return ''
+  }
 
-const customValueFormatter = (value: number) => String(formatSize(value))
+  return String(formatSize(value))
```

Comment:
this cast is not necessary
```suggestion
  return formatSize(value)
```